draft-ietf-rtcweb-jsep.xml

```diff
@@ -3353,7 +3353,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             "ice-lite" attribute MUST NOT be specified.</t>
 
             <t>If the RTP/RTCP multiplexing policy is "require", each m=
-            section MUST contain an "a=rtcp-mux" attribute.</t>
+            section MUST contain an "a=rtcp-mux" attribute. If an
+            "m=" section contains the "a=rtcp-mux-only" attribute
```

Comment:
Assuming I am reading this document correctly, our primary concern here is remote SDP.

So:
1. If we have set require then there better be a=rtcp-mux (or we will reject)
2. If the other side said a=rtcp-mux-only but not a=rtcp-mux then this is bogus SDP and we should reject.

Maybe #1 doesn't go here....

Local SDP doesn't need semantics verification because we do comparison and it's right by construction.